fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/storage/Storage.java

```diff
@@ -168,8 +165,9 @@ default long simulateExtract(T resource, long maxAmount, @Nullable TransactionCo
 	 * @return An iterable over the contents of this storage.
 	 * @see #iterator
 	 */
+	@SuppressWarnings("unchecked")
 	default Iterable<StorageView<T>> iterable(TransactionContext transaction) {
```

Comment:
shouldn't this use `? extends StorageView<T>` too?

Reply:
I'm not sure, I guess it could? I was thinking that it might be a more breaking change, and I'm not too sure that there is a benefit to it. The point was mainly to allow returning a list of something extending `StorageView<T>` in `iterator`.

Reply:
I think that argument applies a little more directly to iterable() than iterator() when it's backed by e.g. a List? Relying on the default impl is not the most efficient at least.

Reply:
Good point, indeed! I changed it and it didn't break any call site besides the default impl, so it shouldn't be too breaking.